pkg/videoproto

```diff
@@ -1,12 +1,13 @@
-[deps]
-util-macros
+[mirrors]
+https://www.x.org/archive//individual/proto/videoproto-2.3.3.tar.gz
```

Comment:
you're kinda inconsistent with http vs https, http is preferable for busybox reasons, and somehow this double slash makes me wonder whether the url is correct.
does the old base url http://xorg.freedesktop.org/releases/individual/ still work ? if so we might add a second mirror here and there so the other URL doesnt get lost in commit history. it's always good to know alternatives.